scrapy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: scrapy
+  version: "1.0.3"
```

Comment:
should we include a packaging version?

Just in case later we package `service_identity` and want to repackage Scrapy 1.0.3 with service_identity as dependency.